diff -Nur ipxripd.orig/ipxkern.c ipxripd.chng/ipxkern.c
--- ipxripd.orig/ipxkern.c	1996-02-01 17:56:07.000000000 +0100
+++ ipxripd.chng/ipxkern.c	2005-07-07 16:37:01.000000000 +0200
@@ -51,7 +51,14 @@
 	FILE *ipx_route;
 	char buf[512];
 
-	ipx_route = fopen("/proc/net/ipx_route", "r");
+	if (access("/proc/net/ipx/route", R_OK) == 0)
+		ipx_route = fopen("/proc/net/ipx/route", "r");
+	else if (access("/proc/net/ipx_route", R_OK) == 0)
+		ipx_route = fopen("/proc/net/ipx_route", "r");
+	else {
+		sprintf(ipx_err_string, "Can't access /proc/net/ipx_route");
+		return -1;
+	}
 
 	if (ipx_route == NULL)
 	{
@@ -109,7 +116,14 @@
 	FILE *ipx_ifc;
 	char buf[512];
 
-	ipx_ifc = fopen("/proc/net/ipx_interface", "r");
+	if (access("/proc/net/ipx/interface", R_OK) == 0)
+		ipx_ifc = fopen("/proc/net/ipx/interface", "r");
+	else if (access("/proc/net/ipx_interface", R_OK) == 0)
+		ipx_ifc = fopen("/proc/net/ipx_interface", "r");
+	else {
+		sprintf(ipx_err_string, "Can't access /proc/net/ipx_interface");
+		return -1;
+	}
 
 	if (ipx_ifc == NULL)
 	{
